README.md

```diff
@@ -241,6 +241,7 @@ The TZ (timezone) environment variable is set to 'Europe/London' in `start.js`.
 | E2E_SUPPLIER_USERNAME | Supplier user username used for acceptance testing | |
 | E2E_SUPPLIER_PASSWORD | Supplier user password used for acceptance testing | |
 | LOCATIONS_BATCH_SIZE | Maximum number of location IDs to send in one request when requesting moves for all locations | 40 |
+| FEATURE_FLAG_PERSON_ESCORT_RECORD | Set to `true` to enable display of the Person Escort Record feature | |
```

Comment:
A word of caution. The current code works for any _truthy_ value. Since env vars are read as strings `false` (as a string) is a truthy value and will enable the feature.

```bash
❯❯❯ TEST=false node
Welcome to Node.js v14.4.0.
Type ".help" for more information.
> console.log(process.env.TEST)
false
undefined
> ({}).toString.call(process.env.TEST)
'[object String]'
> process.env.TEST ? 'is truthy' :  'is falsy'
'is truthy'
```